PYME/Acquire/Hardware/driftTracking.py

```diff
@@ -343,7 +343,8 @@ def tick(self, **kwargs):
             self.history.append((time.time(), dx, dy, dz, cCoeff, self.corrRef, self.piezo.GetOffset(), self.piezo.GetPos(0)))
             eventLog.logEvent('PYME2ShiftMeasure', '%3.4f, %3.4f, %3.4f' % (dx, dy, dz))
             
-            self.lockActive = self.lockFocus and (cCoeff > .5*self.corrRef)

```

Comment:
What motivated this change?

Some form of lock checking code is definitely required to prevent runaway when the system falls out of lock (otherwise any error condition can end up with the drift correction driving the piezo right to one end of it's travel and, at the very least, completely dropping the focus, but potentially also loosing immersion oil etc ...). 

Lock can be lost when:
- there is too much lateral drift (causing calibration and true transmitted images to loose correlation)
- there are glitches in the drift tracking camera and/or illumination (the current logic seemlessly recoverers from these - i.e. when the correlation comes back, the lock continues).
- the system is bumped (or similar) resulting in a sudden position change to outside the calibration region
- your z-stack goes outside the calibration region.

It's imperative to detect and handle these - The current correlation coefficient way is fairly crude, but can't be removed without an effective and tested replacement.